sympy/physics/control/lti.py

```diff
@@ -1259,3 +1453,261 @@ def _eval_rewrite_as_TransferFunction(self, num, den, **kwargs):
 
     def __neg__(self):
         return Feedback(-self.num, self.den)
+
+
+class TransferFunctionMatrix(Basic):
```

Comment:
Please add a class docstring

Reply:
Yeah, will add the docstrings by tonight.